docs/install.md

```diff
@@ -424,6 +432,145 @@ Out-of-the-box, Tekton Pipelines Controller is configured for relatively small-s
 
 You can create a custom release of Tekton Pipelines by following and customizing the steps in [Creating an official release](https://github.com/tektoncd/pipeline/blob/main/tekton/README.md#create-an-official-release). For example, you might want to customize the container images built and used by Tekton Pipelines.
 
+## Verify Tekton Pipelines Release
+
+> We will refine this process over time to be more streamlined. For now, please follow the steps listed in this section
+to verify Tekton pipeline release.
+
+Tekton Pipeline's images are being signed by [Tekton Chains](https://github.com/tektoncd/chains) since [0.27.1](https://github.com/tektoncd/pipeline/releases/tag/v0.27.1). You can verify the images with
+`cosign` using the [Tekton's public key](https://raw.githubusercontent.com/tektoncd/chains/main/tekton.pub).
+
+With Go 1.16+, you can install `cosign` by running:
+
+```shell
+go install github.com/sigstore/cosign/cmd/cosign@latest
+```
+
+Before, you can verify Tekton Pipeline images, collect the list of images created as part of the release. You can search
+the transparency log and find provenance of the released image using `rekor-cli`.
+
+Retrieve the `rekor-cli` by running:
+
+```shell
+go install -v github.com/sigstore/rekor/cmd/rekor-cli@latest
+```
+
+Now, get the digest from the `release.yaml`, for example, collecting the digest for the release `v0.28.0`:
+
+```shell
+curl -s https://storage.googleapis.com/tekton-releases/pipeline/previous/v0.28.0/release.yaml | grep github.com/tektoncd/pipeline/cmd/controller:v0.28.0 | awk -F"github.com/tektoncd/pipeline/cmd/controller:v0.28.0@" '{print $2}'
+```
+
+which results in:
+
+```shell
+sha256:6fd8fe1d19933c62efe0195a05e2c93b5237739dc68c398843dfa2dd22fcb91d
+```
+
+Search the transparency log with the digest just collected:
+
+```shell
+rekor-cli search --sha sha256:6fd8fe1d19933c62efe0195a05e2c93b5237739dc68c398843dfa2dd22fcb91d
+```
+
+which results in:
+
+```shell
+Found matching entries (listed by UUID):
+eb69098b520a93ce140f4facfa596f4947ae7158910531d4882e466c24faec12
+257d788c3e8f3937e7085de5571ffd78d0156461eb7ca67baa9e9e0cdc7b4dc1
+```
+
+Tekton Chains generates provenance based the custom format in which the `subject` holds the list of artifacts which were
```

Comment:
based -> based on

Reply:
Maybe include a link to the provenance spec https://github.com/tektoncd/chains/blob/main/PROVENANCE_SPEC.md